Web/Edubase.Web.UI/Models/EstablishmentDetailViewModel.cs

```diff
@@ -219,5 +219,16 @@ public bool ShowFinancialBenchmarking
         }
 
         public TabWarningsModel TabWarnings { get; set; }
+        public string ClosedStatusMessage
+        {
+            get
+            {
+                var time = Establishment.CloseDate?.ToString("d MMMM yyyy");
```

Comment:
Minor nitpick - this variable contains the date string, not a time

Reply:
Thanks, that's been changed